src/app/nav.module.css

```diff
@@ -0,0 +1,203 @@
+@media (max-width: 768px) {
+  .navRedBar {
+    display: none;
+  }
+
+  .navBar {
+    display: none;
+  }
+
+  .closeIconContainer {
+    position: absolute;
+  }
+
+  .navMobileBar {
+    width: 500px;
+    min-height: 550px;
```

Comment:
Thats a lot of height for the mobile nav bar. Maybe just let it go full height of the screen?